### ISSUE TITLE
Support footer link for beian

### ISSUE THREAD

**Issue by hustjiangtao** [vuejs/vuepress#2087]
#### What problem does this feature solve?

Add footer link for beian in China.

#### What does the proposed API look like?

Just add a <a> link in footer container which can config by  YAML front matter.

#### Are you willing to work on this yourself?

No.It will be a necessary in China from now on, so it's a better idea to solve it in a formal way.

**Comment by ludanxer** [comment]
@hustjiangtao I don't think that the particular footer format for Chinese website would be supported, but allow users to add a unified footer for the whole website may be quite handy.

For now, I think you can try to use [theme inheritance](https://vuepress.vuejs.org/theme/inheritance.html#theme-inheritance) like the [official website](https://github.com/vuejs/vuepress/tree/master/packages/%40vuepress/theme-vue) to add a footer in every page(`slot` is temporarily broken, and will be available in the next release). If you are using the HomePage provided by VuePress, you will need to manually write a footer for that too.

---

For those who are not familiar with Chinese website rules, `备案/beian` is quite hard to translate. Basically, every website deployed in China need to register at a certain government department and get a registration number, and the website is supposed to put that number at the bottom of every web page with a certain format, like a second line in the image below:
![image](https://user-images.githubusercontent.com/27280733/71315652-83ec7a80-249d-11ea-8bfa-e6183f56d7d9.png)

**Comment by tarvos21** [comment]
I also hope that this feature would be added to VuePress, thanks!

**Comment by qcyblm** [comment]
> ## Feature request
> #### What problem does this feature solve?
> Add footer link for beian in China.
> 
> #### What does the proposed API look like?
> Just add a link in footer container which can config by YAML front matter.
> 
> #### How should this be implemented in your opinion?
> #### Are you willing to work on this yourself?
> No.It will be a necessary in China from now on, so it's a better idea to solve it in a formal way.

Recommend a theme [@qcyblm/vuepress-theme-vpx](https://github.com/qcyblm/vuepress-theme-vpx)
